src/array.js

```diff
@@ -49,10 +49,13 @@ TransisArray.of = function() {
 // Public: Creates a new `Transis.Array` from the given array-like object. Useful for converting a
 // regular array to a `Transis.Array`.
 //
-// a - An array-like object.
+// a     - An array-like object.
+// mapFn - Map function to call on every element of the array (optional).
 //
 // Returns a new `Transis.Array`.
-TransisArray.from = function(a) { return TransisArray.of.apply(null, a); };
+TransisArray.from = function(a, mapFn) {
+  return TransisArray.of.apply(null, mapFn ? a.map(a => mapFn(a)) : a);
```

Comment:
@burrows Thanks, that makes sense. My latest commit updates based on your comments. I tried to add a spec to test that mapFn is called with the correct 'this', but was having trouble with the spies. I'll have to spend some more time with it if you think we should add that spec. 68799af74bafc7bc6a4a28178352096152f2018c